pandas/core/common.py

```diff
@@ -1857,23 +1857,84 @@ def _maybe_box_datetimelike(value):
 
 _values_from_object = lib.values_from_object
 
-
-def _possibly_convert_objects(values,
-                              datetime=True,
-                              numeric=True,
-                              timedelta=True,
-                              coerce=False,
+# TODO: Remove in 0.18 or 2017, which ever is sooner
+def _possibly_convert_objects(values, convert_dates=True,
```

Comment:
Fair enough -- but there's not any other way to hook directly into pandas' object array of datetimes -> datetime64 conversion. Might be worth thinking about making such an API at some point.

On Wed, Sep 23, 2015 at 5:58 PM, Jeff Reback <notifications@github.com>
wrote:

>> @@ -1857,23 +1857,84 @@ def _maybe_box_datetimelike(value):
>>  
>>  _values_from_object = lib.values_from_object
>>  
>> -
>> -def _possibly_convert_objects(values,
>> -                              datetime=True,
>> -                              numeric=True,
>> -                              timedelta=True,
>> -                              coerce=False,
>> +# TODO: Remove in 0.18 or 2017, which ever is sooner
>> +def _possibly_convert_objects(values, convert_dates=True,
> @shoyer I don't think you should be using this internal function (I saw you removed it). and instead use the public ``.convert_objects``
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/pydata/pandas/pull/11173/files#r40276398